webapp/src/components/link_tooltip/link_tooltip.jsx

```diff
@@ -0,0 +1,168 @@
+                    <Octicon
+                        icon={iconType}
+                        size='small'
+                        verticalAlign='middle'
+                    />
+                </span>
+            );
+            break;
+        }
+        return icon;
+    };
+
+    if (data) {
+        let date = new Date(data.created_at);
+        date = date.toDateString();
+
+        return (
+            <div className='github-tooltip'>
+                <div className='github-tooltip box github-tooltip--large github-tooltip--bottom-left p-4'>
+                    <div className='header mb-1'>
+                        <a
+                            title={data.repo}
+                            href={href}
+                        >{data.repo}</a>&nbsp;on&nbsp;<span>{date}</span>
+                    </div>
+
+                    <div className='body d-flex mt-2'>
+                        <span className='pt-1 pb-1 pr-2'>
+                            { getIconElement() }
+                        </span>
+
+                        {/* info */}
+                        <div className='tooltip-info mt-1'>
+                            <a href={href}>
+                                <h5 className='mr-1'>{data.title}</h5>
+                                <span>#{data.number}</span>
+                            </a>
+                            <div className='markdown-text mt-1 mb-1'>
+                                <ReactMarkdown
+                                    source={data.body}
+                                    disallowedTypes={['heading']}
+                                    linkTarget='_blank'
+                                />
+                            </div>
+
+                            {/* base <- head */}
+                            {data.type === 'pull' && (
+                                <div className='base-head mt-1 mr-3'>
+                                    <span
+                                        title={data.base.ref}
+                                        className='commit-ref'
+                                        style={{maxWidth: '140px'}}
+                                    >{data.base.ref}
+                                    </span>
+                                    <span className='mx-1'>←</span>
+                                    <span
+                                        title={data.head.ref}
+                                        className='commit-ref'
+                                    >{data.head.ref}
+                                    </span>
+                                </div>
+                            )}
+
+                            <div className='see-more mt-1'>
+                                <a
+                                    href={href}
+                                    target='_blank'
+                                    rel='noopener noreferrer'
+                                >See more</a>
+                            </div>
```

Comment:
2/5: It's quite common to have this kind of links put at the end of a unit. Hence I'm wondering if the link should be put after the labels.

Reply:
hey @hanzei, I put the link there because @asaadmahmood told me to do it like that. Check his comment [https://github.com/mattermost/mattermost-plugin-github/pull/258#discussion_r423559012](here). But if you prefer me to put it below the labes, I can do it! Let me know your thoughts!

Reply:
Good point, I missed that comment. Then I'm fine with how it's is.